### PR TITLE
add config import.memory-use-ratio

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12279
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1601,6 +1601,16 @@ Configuration items related to TiDB Lightning import and BR restore.
 + Default value: `8`
 + Minimum value: `1`
 
+### `memory-use-ratio` <span class="version-mark">New in v6.5.0</span>
+
+> **Note:**
+>
+> In versions earlier than v6.5.0, point-in-time recovery (PITR) only supports restoring data by downloading backup files to a local directory. Starting from v6.5.0, PITR supports directly accessing backup log files and restoring data in memory. This configuration item specifies the memory usage ratio for PITR in TiKV.
```
Comment (joccau) [inline tikv-configuration-file.md:1608]:
```suggestion
> In versions earlier than v6.5.0, point-in-time recovery (PITR) only supports restoring data by downloading backup files to a local directory. Starting from v6.5.0, PITR supports directly accessing backup log files in memory and restoring data. This configuration item specifies the memory usage ratio for PITR in TiKV.
```

**Review by joccau — COMMENTED** [review]
rest LGTM

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1601,6 +1601,16 @@ Configuration items related to TiDB Lightning import and BR restore.
...
 
+### `memory-use-ratio` <span class="version-mark">New in v6.5.0</span>
+
+> **Note:**
+>
+> In versions earlier than v6.5.0, point-in-time recovery (PITR) only supports restoring data by downloading backup files to a local directory. Starting from v6.5.0, PITR supports directly accessing backup log files in memory and restoring data. This configuration item specifies the memory usage ratio for PITR in TiKV.
+
++ The ratio of memory available for PITR to the total memory of TiKV.
++ Value range: [0.0, 0.5]
++ Default value: `0.3`, which means that 30% of the system memory is available for PITR. When the value is `0.0`, PITR is performed by downloading log files to a local directory.
```
Comment (hfxsd) [inline tikv-configuration-file.md:1612]:
```suggestion
+ Starting from v6.5.0, PITR supports directly accessing backup log files in memory and restoring data. This configuration item specifies the ratio of memory available for PITR to the total memory of TiKV.
+ Value range: [0.0, 0.5]
+ Default value: `0.3`, which means that 30% of the system memory is available for PITR. When the value is `0.0`, PITR is performed through downloading log files to a local directory.

> **Note:**
>
> In versions earlier than v6.5.0, point-in-time recovery (PITR) only supports restoring data by downloading backup files to a local directory. 
```

**Review by hfxsd — APPROVED** [review]

**Review by joccau — APPROVED** [review]
LGTM

**Pull request merged**